samples/nrf9160/modem_shell/README.rst

```diff
@@ -44,10 +44,31 @@ Enable and configure eDRX::
 
    link edrx -e -m --edrx_value 0010 --ptw 0010
 
-Enable and configure PSM::
+Subscribe for modem TAU and sleep notifications, enable and configure PSM::
 
+   link tau --subscribe
+   link msleep --subscribe
    link psm -e --rptau 01100100 --rat 00100010
 
+Disable autoconnect::
+
+   link nmodeauto --disable
+   
+   Reboot the DK and it remains in pwroff functional mode and link is not connected.
+
+Link settings::
+
+   Some of the link subcommand parameters are stored to settings and are persistent between sessions.
```

Comment:
Do you want to have this in the code snippet format even though there is only text?

Reply:
No, we'll tune it

Reply:
Moving this to Link description in my next change